index.html

```diff
@@ -1036,7 +1041,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.alexcg.net"><i class="icon fa fa-external-link"></i></a>
-											<a target="default" href="https://twitter.com/alexcg"><i class="icon social_twitter"></i></a>
+											<a target="default" href="https://github.com/alexcg1"><i class="icon social_twitter"></i></a>
```

Comment:
Github link in twitter icon can be misleading, please don't remove the twitter icon, add github icon separately.